packages/core/src/file-context.ts

```diff
@@ -52,6 +53,31 @@ export class FileContext {
     return this._ignoredChecks;
   }
 
+  get ignoredChecksByLine(): Map<number, string[]> {
+    const sourceFile = ts.createSourceFile(this.filename, this.content, ts.ScriptTarget.ES5, true);
```

Comment:
This already exists as `source`...
```suggestion
    const sourceFile = this.source;
```

Reply:
Fixed it in the next iteration.